libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -184,6 +184,21 @@
         LOGIN_INSERTED_INVALID_URL,
         LOGIN_AUTOFILL_CREDENTIALS_FILLED,
         LOGIN_AUTOFILL_CREDENTIALS_UPDATED,
+        LOGIN_PROLOGUE_VIEWED,
+        LOGIN_EMAIL_FORM_VIEWED,
+        LOGIN_EMAIL_RETRY_VIEWED,
+        LOGIN_MAGIC_LINK_OPEN_FORM_VIEWED,
+        LOGIN_MAGIC_LINK_REQUEST_FORM_VIEWED,
+        LOGIN_PASSWORD_FORM_VIEWED,
+        LOGIN_PASSWORD_RETRY_VIEWED,
```

Comment:
Same as for the LOGIN_URL_RETRY_VIEWED case, this is most probably not a different screen so, no  new "VIEWED" event would be emitted.

Let's introduce a separate set of events here. Perhaps even group these under a single event: "LOGIN_INPUT_ERROR" with the specific error being a property of this event. WDYT?